tests/ui/empty_line_after_outer_attribute.stderr

```diff
@@ -2,7 +2,7 @@ error: Found an empty line after an outer attribute. Perhaps you forgot to add a
  --> $DIR/empty_line_after_outer_attribute.rs:5:1
   |
 5 | / #[crate_type = "lib"]
-6 | |
+6 | | 
```

Comment:
It looks to me like Travis is unhappy with these new spaces. I'll have a look later today. Sorry that you ran into this. We should've detected this on master and fixed it independently

Reply:
No problem, I've got to mess around with more rust than I have in a while and get some experience working with travis and github that I didn't have. I appreciate that you have been patient while I worked on this. Figuring out how to work with these large sets of rust code and working with github has been good. Let me know if there's something I can look into or check on my side.